hack/ds_test.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+set -e
+
+# Create the namespace for the HCO
+oc create ns kubevirt-hyperconverged
+
+# Create an OperatorGroup
+cat <<EOF | oc create -f -
+apiVersion: operators.coreos.com/v1alpha2
+kind: OperatorGroup
+metadata:
+  name: hco-operatorgroup
+  namespace: kubevirt-hyperconverged
+EOF
+
+# Create a Catalog Source
+cat <<EOF | oc create -f -
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: hco-catalogsource
+  namespace: openshift-operator-lifecycle-manager
+spec:
+  sourceType: grpc
+  image: brew-pulp-docker01.web.prod.ext.phx2.redhat.com:8888/container-native-virtualization/hyperconverged-cluster-operator:v2.0.0-4
```

Comment:
this image isn't setup to use grpc to provide a CatalogSource.  We can use the hco-registry image in `deploy/converged/Dockerfile` to make the grpc bundle and point at your `downstream` CSV.  The would be hosted upstream temporarily until we sort out quay.

Reply:
done